workspaces/services/guacamole/terraform/locals.tf

```diff
@@ -7,11 +7,7 @@ resource "random_string" "unique_id" {
 locals {
   service_id                   = random_string.unique_id.result
   service_resource_name_suffix = "${var.tre_id}-ws-${var.workspace_id}-svc-${local.service_id}"
+  webapp_name                  = "guacamole-${local.service_resource_name_suffix}"
```

Comment:
Why put webapp name in locals and not just state it in the web_app.tf? 
If we take the azureml service for reference it's not the convention there

Reply:
It's just a "shorthand" since we keep using it in many places then I define it once in locals and then is able to reference it everywhere I need to.